File: src/URI/URI_1006.java
package URI;
import java.util.Locale;
import java.util.Scanner;

public class URI_1006 {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double nota1, nota2, nota3, media;
		
		nota1 = sc.nextDouble();
		nota2 = sc.nextDouble();
		nota3 = sc.nextDouble();
		media = (nota1 * 2 + nota2 * 3 + nota3 * 5) / 10;
		
		System.out.printf("MEDIA = %.1f%n", media);
		sc.close();

	}

}

File: kotlin/URI_1006.kt
import java.util.*

const val pesoA = 2
const val pesoB = 3
const val pesoC = 5

fun main(args: Array<String>) {
    Locale.setDefault(Locale.US)
    val sc = Scanner(System.`in`)
    val a = sc.nextDouble()
    val b = sc.nextDouble()
    val c = sc.nextDouble()

    val media = {x : Double, y: Double, z: Double -> (x * pesoA + y * pesoB + z * pesoC) / 10.0}
    println(String.format("MEDIA = %.1f", media(a,b,c)))
}